nrfu_tests/test_system_hardware_coredump.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2023 Arista Networks, Inc.  All rights reserved.
+# Arista Networks, Inc. Confidential and Proprietary.
+
+"""
+Test case to verify that core dump files are not found
+"""
+
+import pytest
+from pyeapi.eapilib import EapiError
+from vane.logger import logger
+from vane.config import dut_objs, test_defs
+from vane import tests_tools
+
+TEST_SUITE = "nrfu_tests"
+
+
+@pytest.mark.nrfu_test
+@pytest.mark.system
+class CoreDumpFilesTests:
+    """
+    Test case to verify that core dump files are not found
+    """
+
+    dut_parameters = tests_tools.parametrize_duts(TEST_SUITE, test_defs, dut_objs)
+    test_duts = dut_parameters["test_hardware_core_dump"]["duts"]
+    test_ids = dut_parameters["test_hardware_core_dump"]["ids"]
+
+    @pytest.mark.parametrize("dut", test_duts, ids=test_ids)
+    def test_hardware_core_dump(self, dut, tests_definitions):
+        """
+        TD: Test case to verify that core dump files are not found
+        Args:
+            dut(dict): details related to a particular DUT
+            tests_definitions(dict): test suite and test case parameters.
+        """
+
+        tops = tests_tools.TestOps(tests_definitions, TEST_SUITE, dut)
+        self.output = ""
+        tops.actual_output = {}
+        tops.output_msg = "Core dump files are not found."

```

Comment:
can we update? please correct me if I am wrong.
```suggestion
        tops.output_msg = "Core dump files are not found on the device."

```